libtransmission/peer-io.h

```diff
@@ -93,7 +93,7 @@ struct tr_peerIo
 
     // Changed to non-owning pointer temporarily till tr_peerIo becomes C++-constructible and destructible
     // TODO: change tr_bandwidth* to owning pointer to the bandwidth, or remove * and own the value
-    struct tr_bandwidth* bandwidth;
+    struct Bandwidth* bandwidth;
```

Comment:
was it intentional to leave the `struct` here and below even after the forward declaration?

Reply:
The renames are done using CLion's built-in refactoring, it tries to guarantee that all enabled code was changed correctly (by enabled I mean the code not disabled with `#ifdefs`) and I avoid changing text and comments automatically it always creates a mess. Instead i try to remember to also go through the comments referring to renamed data types and correct those too. This is quite effortless task, but it took me about 4 tries resetting git state every time till I got it right.

For naming if there is a way to produce non-fatal warnings i'd be all up for it. For now I have configured the preferred naming style as a hint in CLion and react to it every time i see it, there isn't an automated check or a warning on that, so all the old style code does not light up like a christmas tree.

`struct` I have removed it in bandwidth.cc, but not in outer files - next time i meet it, i will edit it out. Did not really hunt for every use of `Bandwidth`, but i can for this pull request to make it look good.

Reply:
Edited those extraneous `structs` out in the following commit

Reply:
For Clang-tidy that's also integrated into CLion if i allow it, it will highlight entire source like a Christmas tree.
It is quite convenient, that IDE integration.
![bild](https://user-images.githubusercontent.com/288724/136677351-4d8d9ab7-7cc0-49a1-b78a-ff6ee5b3dc2a.png)